frontend/src/components/form/BaseFileUpload.vue

```diff
@@ -84,7 +84,7 @@ const submit = async () => {
   <div class="flex flex-col items-center gap-2">
     <div class="py-8">
       <img
-        class="h-24"
+        class="h-24 circular-image"
```

Comment:
good job changing the image to circle but again the image is too small 
![image](https://github.com/sain0136/myrotary-projects/assets/55252053/e0f57a93-8431-47a7-a497-4d73511ba330)

like this size  in the image make sure it looks good as well on all screen types 
![image](https://github.com/sain0136/myrotary-projects/assets/55252053/ffad342b-b1d8-4ccf-9b2f-086f206b9016)